Gemfile

```diff
@@ -41,4 +43,6 @@ group :development, :test do
   gem 'rspec-rails', '~> 3.6.0' 
   gem 'factory_bot_rails'
   gem 'faker'
+  gem 'test-unit', '~> 3.0'
+  gem 'will_paginate'
 end
```

Comment:
A minor nitpick - you may need to set your editor to insert newlines at the end of file on save